### ISSUE TITLE
无法训练faq_system train.py脚本，直接中断

### ISSUE THREAD

**Issue by konghehan** [PaddlePaddle/PaddleNLP#2344]
- 版本、环境信息
1）PaddleNLP和PaddlePaddle版本：PaddleNLP 2.3.1，PaddlePaddle2.3.0
2）系统环境：centos7，python3.8
- 复现信息：如为报错，请给出复现环境、复现步骤

详细信息如下：
python3 -u -m paddle.distributed.launch --gpus '0' \
>     train.py \
>     --device gpu \
>     --save_dir ./checkpoints/ \
>     --batch_size 64 \
>     --learning_rate 5E-5 \
>     --epochs 3 \
>     --save_steps 50 \
>     --max_seq_length 64 \
>     --dropout 0.2 \
>     --output_emb_size 256 \
>     --dup_rate 0.3 \
>     --train_set_file "./data/train.csv"
/usr/local/python3/lib/python3.8/site-packages/setuptools/depends.py:2: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
LAUNCH INFO 2022-05-29 18:51:54,451 -----------  Configuration  ----------------------
LAUNCH INFO 2022-05-29 18:51:54,451 devices: None
LAUNCH INFO 2022-05-29 18:51:54,452 elastic_level: -1
LAUNCH INFO 2022-05-29 18:51:54,452 elastic_timeout: 30
LAUNCH INFO 2022-05-29 18:51:54,452 gloo_port: 6767
LAUNCH INFO 2022-05-29 18:51:54,452 host: None
LAUNCH INFO 2022-05-29 18:51:54,452 job_id: default
LAUNCH INFO 2022-05-29 18:51:54,452 legacy: False
LAUNCH INFO 2022-05-29 18:51:54,452 log_dir: log
LAUNCH INFO 2022-05-29 18:51:54,452 log_level: INFO
LAUNCH INFO 2022-05-29 18:51:54,452 master: None
LAUNCH INFO 2022-05-29 18:51:54,452 max_restart: 3
LAUNCH INFO 2022-05-29 18:51:54,452 nnodes: 1
LAUNCH INFO 2022-05-29 18:51:54,452 nproc_per_node: None
LAUNCH INFO 2022-05-29 18:51:54,452 rank: -1
LAUNCH INFO 2022-05-29 18:51:54,452 run_mode: collective
LAUNCH INFO 2022-05-29 18:51:54,452 server_num: None
LAUNCH INFO 2022-05-29 18:51:54,452 servers:
LAUNCH INFO 2022-05-29 18:51:54,452 trainer_num: None
LAUNCH INFO 2022-05-29 18:51:54,452 trainers:
LAUNCH INFO 2022-05-29 18:51:54,452 training_script: 0
LAUNCH INFO 2022-05-29 18:51:54,452 training_script_args: ['train.py', '--device', 'gpu', '--save_dir', './checkpoints/', '--batch_size', '64', '--learning_rate', '5E-5', '--epochs', '3', '--save_steps', '50', '--max_seq_length', '64', '--dropout', '0.2', '--output_emb_size', '256', '--dup_rate', '0.3', '--train_set_file', './data/train.csv']
LAUNCH INFO 2022-05-29 18:51:54,452 with_gloo: 0
LAUNCH INFO 2022-05-29 18:51:54,452 --------------------------------------------------
LAUNCH WARNING 2022-05-29 18:51:54,452 Compatible mode enable with args ['--gpus']
-----------  Configuration Arguments -----------
backend: auto
cluster_topo_path: None
elastic_pre_hook: None
elastic_server: None
enable_auto_mapping: False
force: False
heter_devices:
heter_worker_num: None
heter_workers:
host: None
http_port: None
ips: 127.0.0.1
job_id: None
log_dir: log
np: None
nproc_per_node: None
rank_mapping_path: None
run_mode: None
scale: 0
server_num: None
servers:
training_script: 0
training_script_args: ['train.py', '--device', 'gpu', '--save_dir', './checkpoints/', '--batch_size', '64', '--learning_rate', '5E-5', '--epochs', '3', '--save_steps', '50', '--max_seq_length', '64', '--dropout', '0.2', '--output_emb_size', '256', '--dup_rate', '0.3', '--train_set_file', './data/train.csv']
worker_num: None
workers:
------------------------------------------------
WARNING 2022-05-29 18:51:54,453 launch.py:714] launch start with CPUONLY mode
WARNING 2022-05-29 18:51:54,453 launch.py:714] launch start with CPUONLY mode
launch train in CPU mode
INFO 2022-05-29 18:51:54,454 launch_utils.py:558] Local start 1 processes. First process distributed environment info (Only For Debug):
    +=======================================================================================+
    |                        Distributed Envs                      Value                    |
    +---------------------------------------------------------------------------------------+
    |                       PADDLE_TRAINER_ID                        0                      |
    |                 PADDLE_CURRENT_ENDPOINT                 127.0.0.1:38672               |
    |                     PADDLE_TRAINERS_NUM                        1                      |
    |                PADDLE_TRAINER_ENDPOINTS                 127.0.0.1:38672               |
    |                     PADDLE_RANK_IN_NODE                        0                      |
    |                 PADDLE_LOCAL_DEVICE_IDS                                               |
    |                 PADDLE_WORLD_DEVICE_IDS                                               |
    +=======================================================================================+

INFO 2022-05-29 18:51:54,454 launch_utils.py:558] Local start 1 processes. First process distributed environment info (Only For Debug):
    +=======================================================================================+
    |                        Distributed Envs                      Value                    |
    +---------------------------------------------------------------------------------------+
    |                       PADDLE_TRAINER_ID                        0                      |
    |                 PADDLE_CURRENT_ENDPOINT                 127.0.0.1:38672               |
    |                     PADDLE_TRAINERS_NUM                        1                      |
    |                PADDLE_TRAINER_ENDPOINTS                 127.0.0.1:38672               |
    |                     PADDLE_RANK_IN_NODE                        0                      |
    |                 PADDLE_LOCAL_DEVICE_IDS                                               |
    |                 PADDLE_WORLD_DEVICE_IDS                                               |
    +=======================================================================================+

INFO 2022-05-29 18:51:54,454 launch_utils.py:563] details about PADDLE_TRAINER_ENDPOINTS can be found in log/endpoints.log, and detail running logs maybe found in log/workerlog.0
INFO 2022-05-29 18:51:54,454 launch_utils.py:563] details about PADDLE_TRAINER_ENDPOINTS can be found in log/endpoints.log, and detail running logs maybe found in log/workerlog.0
launch proc_id:8005 idx:0
/usr/bin/python3: can't open file '0': [Errno 2] No such file or directory
INFO 2022-05-29 18:52:01,476 launch_utils.py:343] terminate all the procs
INFO 2022-05-29 18:52:01,476 launch_utils.py:343] terminate all the procs
ERROR 2022-05-29 18:52:01,476 launch_utils.py:640] ABORT!!! Out of all 1 trainers, the trainer process with rank=[0] was aborted. Please check its log.
ERROR 2022-05-29 18:52:01,476 launch_utils.py:640] ABORT!!! Out of all 1 trainers, the trainer process with rank=[0] was aborted. Please check its log.
INFO 2022-05-29 18:52:05,480 launch_utils.py:343] terminate all the procs
INFO 2022-05-29 18:52:05,480 launch_utils.py:343] terminate all the procs
INFO 2022-05-29 18:52:05,480 launch.py:402] Local processes completed.
INFO 2022-05-29 18:52:05,480 launch.py:402] Local processes completed.

**Comment by w5688414** [comment]
```
/usr/bin/python3: can't open file '0': [Errno 2] No such file or directory
```
看上去是格式问题？0表示使用的显卡是0，请问是运行的bash脚本吗？

> * 版本、环境信息
>   1）PaddleNLP和PaddlePaddle版本：PaddleNLP 2.3.1，PaddlePaddle2.3.0
>   2）系统环境：centos7，python3.8
> * 复现信息：如为报错，请给出复现环境、复现步骤
> 
> 详细信息如下： python3 -u -m paddle.distributed.launch --gpus '0' \
> 
> > ```
> > train.py \
> > --device gpu \
> > --save_dir ./checkpoints/ \
> > --batch_size 64 \
> > --learning_rate 5E-5 \
> > --epochs 3 \
> > --save_steps 50 \
> > --max_seq_length 64 \
> > --dropout 0.2 \
> > --output_emb_size 256 \
> > --dup_rate 0.3 \
> > --train_set_file "./data/train.csv"
> > ```
> 
> ## /usr/local/python3/lib/python3.8/site-packages/setuptools/depends.py:2: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
> import imp
> LAUNCH INFO 2022-05-29 18:51:54,451 ----------- Configuration ----------------------
> LAUNCH INFO 2022-05-29 18:51:54,451 devices: None
> LAUNCH INFO 2022-05-29 18:51:54,452 elastic_level: -1
> LAUNCH INFO 2022-05-29 18:51:54,452 elastic_timeout: 30
> LAUNCH INFO 2022-05-29 18:51:54,452 gloo_port: 6767
> LAUNCH INFO 2022-05-29 18:51:54,452 host: None
> LAUNCH INFO 2022-05-29 18:51:54,452 job_id: default
> LAUNCH INFO 2022-05-29 18:51:54,452 legacy: False
> LAUNCH INFO 2022-05-29 18:51:54,452 log_dir: log
> LAUNCH INFO 2022-05-29 18:51:54,452 log_level: INFO
> LAUNCH INFO 2022-05-29 18:51:54,452 master: None
> LAUNCH INFO 2022-05-29 18:51:54,452 max_restart: 3
> LAUNCH INFO 2022-05-29 18:51:54,452 nnodes: 1
> LAUNCH INFO 2022-05-29 18:51:54,452 nproc_per_node: None
> LAUNCH INFO 2022-05-29 18:51:54,452 rank: -1
> LAUNCH INFO 2022-05-29 18:51:54,452 run_mode: collective
> LAUNCH INFO 2022-05-29 18:51:54,452 server_num: None
> LAUNCH INFO 2022-05-29 18:51:54,452 servers:
> LAUNCH INFO 2022-05-29 18:51:54,452 trainer_num: None
> LAUNCH INFO 2022-05-29 18:51:54,452 trainers:
> LAUNCH INFO 2022-05-29 18:51:54,452 training_script: 0
> LAUNCH INFO 2022-05-29 18:51:54,452 training_script_args: ['train.py', '--device', 'gpu', '--save_dir', './checkpoints/', '--batch_size', '64', '--learning_rate', '5E-5', '--epochs', '3', '--save_steps', '50', '--max_seq_length', '64', '--dropout', '0.2', '--output_emb_size', '256', '--dup_rate', '0.3', '--train_set_file', './data/train.csv']
> LAUNCH INFO 2022-05-29 18:51:54,452 with_gloo: 0
> LAUNCH INFO 2022-05-29 18:51:54,452 --------------------------------------------------
> LAUNCH WARNING 2022-05-29 18:51:54,452 Compatible mode enable with args ['--gpus']
> ----------- Configuration Arguments -----------
> backend: auto
> cluster_topo_path: None
> elastic_pre_hook: None
> elastic_server: None
> enable_auto_mapping: False
> force: False
> heter_devices:
> heter_worker_num: None
> heter_workers:
> host: None
> http_port: None
> ips: 127.0.0.1
> job_id: None
> log_dir: log
> np: None
> nproc_per_node: None
> rank_mapping_path: None
> run_mode: None
> scale: 0
> server_num: None
> servers:
> training_script: 0
> training_script_args: ['train.py', '--device', 'gpu', '--save_dir', './checkpoints/', '--batch_size', '64', '--learning_rate', '5E-5', '--epochs', '3', '--save_steps', '50', '--max_seq_length', '64', '--dropout', '0.2', '--output_emb_size', '256', '--dup_rate', '0.3', '--train_set_file', './data/train.csv']
> worker_num: None
> workers:
> WARNING 2022-05-29 18:51:54,453 launch.py:714] launch start with CPUONLY mode WARNING 2022-05-29 18:51:54,453 launch.py:714] launch start with CPUONLY mode launch train in CPU mode INFO 2022-05-29 18:51:54,454 launch_utils.py:558] Local start 1 processes. First process distributed environment info (Only For Debug): +=======================================================================================+ | Distributed Envs Value | +---------------------------------------------------------------------------------------+ | PADDLE_TRAINER_ID 0 | | PADDLE_CURRENT_ENDPOINT 127.0.0.1:38672 | | PADDLE_TRAINERS_NUM 1 | | PADDLE_TRAINER_ENDPOINTS 127.0.0.1:38672 | | PADDLE_RANK_IN_NODE 0 | | PADDLE_LOCAL_DEVICE_IDS | | PADDLE_WORLD_DEVICE_IDS | +=======================================================================================+
> 
> INFO 2022-05-29 18:51:54,454 launch_utils.py:558] Local start 1 processes. First process distributed environment info (Only For Debug): +=======================================================================================+ | Distributed Envs Value | +---------------------------------------------------------------------------------------+ | PADDLE_TRAINER_ID 0 | | PADDLE_CURRENT_ENDPOINT 127.0.0.1:38672 | | PADDLE_TRAINERS_NUM 1 | | PADDLE_TRAINER_ENDPOINTS 127.0.0.1:38672 | | PADDLE_RANK_IN_NODE 0 | | PADDLE_LOCAL_DEVICE_IDS | | PADDLE_WORLD_DEVICE_IDS | +=======================================================================================+
> 
> INFO 2022-05-29 18:51:54,454 launch_utils.py:563] details about PADDLE_TRAINER_ENDPOINTS can be found in log/endpoints.log, and detail running logs maybe found in log/workerlog.0 INFO 2022-05-29 18:51:54,454 launch_utils.py:563] details about PADDLE_TRAINER_ENDPOINTS can be found in log/endpoints.log, and detail running logs maybe found in log/workerlog.0 launch proc_id:8005 idx:0 /usr/bin/python3: can't open file '0': [Errno 2] No such file or directory INFO 2022-05-29 18:52:01,476 launch_utils.py:343] terminate all the procs INFO 2022-05-29 18:52:01,476 launch_utils.py:343] terminate all the procs ERROR 2022-05-29 18:52:01,476 launch_utils.py:640] ABORT!!! Out of all 1 trainers, the trainer process with rank=[0] was aborted. Please check its log. ERROR 2022-05-29 18:52:01,476 launch_utils.py:640] ABORT!!! Out of all 1 trainers, the trainer process with rank=[0] was aborted. Please check its log. INFO 2022-05-29 18:52:05,480 launch_utils.py:343] terminate all the procs INFO 2022-05-29 18:52:05,480 launch_utils.py:343] terminate all the procs INFO 2022-05-29 18:52:05,480 launch.py:402] Local processes completed. INFO 2022-05-29 18:52:05,480 launch.py:402] Local processes completed.
